api/rpc/storage/storage.go

```diff
@@ -0,0 +1,101 @@
+package storage
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/appcelerator/amp/data/storage"
+	"github.com/golang/protobuf/proto"
+	"golang.org/x/net/context"
+)
+
+const storageRootKey = "storage"
+
+// Server is used to implement storage service
+type Server struct {
+	Store storage.Interface
+}
+
+// Put implements Server API for Put Storage service
+func (s *Server) Put(ctx context.Context, in *PutStorage) (*StorageResponse, error) {
+	//verify if key-value pair does not already exist
```

Comment:
@neha-viswanathan You mentioned you would be removing comments and making some changes before the PR was finished.